src/components/VIcon/VIcon.vue

```diff
@@ -59,14 +59,14 @@ export default defineComponent({
     },
     /**
      * The size of the icon based on tailwind values. Possible values:
-     * 4 - 1rem, 5 - 1.25rem, 6 - 1.5rem.
+     * 4 - 1rem, 5 - 1.25rem, 6 - 1.5rem, 8 - 2rem, 12 - 3rem.
      *
      * @default 6
      */
     size: {
       type: Number,
       default: 6,
-      validator: (val: number) => [4, 5, 6].includes(val),
+      validator: (val: number) => [4, 5, 6, 8, 12].includes(val),
```

Comment:
Instead of this being a prop, the parent should be able to set `h-*` and `w-*` styles on the component. This seems a very contrived way to provide more sizes.

Reply:
Also the new sizes added here, being dynamic, should be added to `tailwind.safelist.txt`.

Reply:
Do you suggest that we scrap the `size` prop altogether, and just use `h-* w-*` classes set by the parent component, with a fallback of `h-6 w-6`?

Reply:
I'm suggesting scrapping the prop in the long term and adding the safelist to approve the PR.

Reply:
I do like the API design of `size` type props, but I think it makes the most sense when standardized across a component library with lots of small, composable ui building blocks. I agree with @dhruvkb's suggestion here.

Reply:
I've removed the prop, and since all classes are not dynamically generated, did not have to add anything to the safelist.

Reply:
I've tried using classes, but then there is a problem with default and overrides:
- if we add `w-6 h-6` by default, then to override them, we would need to add `!` important modifier.
- if we do not add a default, then we would need to add the size classes to _all_ VIcons.

So, I went back to the `size` prop as a cleaner option, and opened an alternative PR: https://github.com/WordPress/openverse-frontend/pull/1928